ui/src/components/Function/RunOutput.tsx

```diff
@@ -1,31 +1,36 @@
 import CodeBlock from '@/components/Code/CodeBlock';
 import renderRunOutput from '@/components/Function/RunOutputRenderer';
-import { FunctionRunStatus, type FunctionRun } from '@/store/generated';
 import { usePrettyJson } from '@/hooks/usePrettyJson';
+import { FunctionRunStatus } from '@/store/generated';
+import { type HistoryNode } from '../TimelineV2/historyParser';
 
 interface RunOutputCardProps {
-  functionRun: Omit<FunctionRun, 'function' |'history' | 'functionID' | 'historyItemOutput'>;
+  status: FunctionRunStatus | HistoryNode['status'];
```

Comment:
Can we pick one or the other? Supporting both status enums/unions leads to more status checking complexity in here